src/localization/fi.json

```diff
@@ -8,9 +8,16 @@
   "header": {
     "hello": "Hei",
     "logout": "Kirjaudu ulos",
-    "home": "Koti",
-    "admin": "Ylläpito",
-    "onCall": "Päivystys"
+    "pages": [
+      "Koti",
+      "Aikapankki",
+      "Lomat",
+      "Päivystys",
+      "Ylläpito"
+    ],
```

Comment:
We don't typically have lists in the localization, as mentioned in a previous comment I think an enum could be an alternative solution